.changes/next-release/feature-AmazonS3TransferManager-21d3a50.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "feature",
+    "category": "Amazon S3 Transfer Manager",
+    "contributor": "",
+    "description": "Add support for etag validation in resumableFileDownload"
```

Comment:
Not sure what the general standards are for these - but maybe add description of the behavior? (something like "restart paused downloads when etag does not match")

Reply:
+1, we don't have a written guidance, but the general rule is that describe the bahvior instead of implementation to make it clear to customers

Reply:
done